util.scad

```diff
@@ -2,12 +2,28 @@
  * Copyright 2018 Mike and Nathan Fairhurst. See LICENSE for licensing.
  */
 
+// Constants specific to this file
+cube_point_rotation = [45, 35.264, 180];
+
+
+// Functions to be used by all parts
+
 function is_number(x) = abs(x)!=undef;
 function sqr(x) = x*x;
 function sqrt(x) = pow(x, 0.5);
 
-// Constants specific to this file
-cube_point_rotation = [45, 35.264, 180];
+// Generate a rotation matrix. You can multiply the result of this function
+// to a 3d vector to rotate it in 3d space as if you'd used [rotate].
+function rotation(rv)
```

Comment:
I might call this `rotation_matrix` so that it's clear it's not a standard primitive.  Not sure I totally grasp why this is needed...